lib/shopify-cli/resources/organizations.rb

```diff
@@ -1,5 +1,5 @@
 module ShopifyCli
-  module Helpers
+  module Resources
     class Organizations
```

Comment:
Would this one be better inside `ShopifyCli::PartnersAPI`? We could just move these methods into that file, rename them a bit. I'm fine either way